client/src/app/pages/applications/analysis-details/AnalysisDetails.tsx

```diff
@@ -45,11 +67,34 @@ export const AnalysisDetails: React.FC = () => {
                 taskId: taskId,
               }),
             },
+            ...(attachmentName
+              ? [
+                  {
+                    title: t("terms.attachments"),
+                  },
+                  {
+                    title: attachmentName,
+                    path: formatPath(Paths.applicationsAnalysisDetails, {
+                      applicationId: applicationId,
+                      taskId: taskId,
+                      attachment: attachmentId,
+                    }),
+                  },
+                ]
+              : []),
           ]}
         />
       </PageSection>
       <PageSection className="simple-task-viewer-container">
-        <SimpleDocumentViewer documentId={Number(taskId)} height="full" />
+        <SimpleDocumentViewer
```

Comment:
Maybe do it like on other pages?

https://github.com/konveyor/tackle2-ui/blob/c2f4f067b14eacf6742e0033e2244ff864d00085/client/src/app/pages/archetypes/archetypes-page.tsx#L362-L366